api/src/Controller/BaseController.php

```diff
@@ -148,6 +155,18 @@ protected function filterOutput(Request $request, $data, $code): array
 
     }
```

Comment:
Could you explain the need for these BaseController changes? 
You want to send a code?

Reply:
The correct code for a successful Create is 201 (unless it's pushed to a queue, in which case it's 202).

The correct code for a successful Delete is 204.

There was no way to send that without this change.

Reply:
Wow so I totally did not know about and of those 20x codes.  Thanks! The rest of the code will have to be updated...

Reply:
Yeah, it's not a huge emergency when we're talking to ourselves and checking only against our own expectations, but I know we're trying to be Good Citizens here. I'm not sure how picky other clients would be; only that 201 means 'Created' and should be what's sent on a Create post. Any OTHER use of POST (say, a form that results in sending e-mail and not creating a record) could return 200, so it's not so much specific to POST as specific to something being created as a result of the action.